bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -2793,7 +2793,7 @@ internal class PlaintextBallotHandle
                 [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
                 protected override bool Free()
                 {
-                    if (IsClosed) return true;
+                    if (IsFreed) return true;
```

Comment:
For the ballot classes, it was thinking that the handle had been closed already and so it never tried to free the unmanaged pointer for it.  Changed to a new variable that can be controlled instead of one from the base class.